src/managers/GuildMemberManager.js

```diff
@@ -127,6 +127,23 @@ class GuildMemberManager extends BaseManager {
     return this._fetchMany(options);
   }
 
+  /**
+   * Search for members in the guild based on a query
+   * @param {Object} options Search options
+   * @property {string} options.query Members whose usernames or nicknames that start with
```

Comment:
Could it be something like this? Taken from `GMM.fetch()`
```suggestion
   * @property {string} options.query Limit search to members with similar usernames
```

Reply:
I was gunna go with that initially, but "similar" could be a bit misleading (e.g: querying "havc" or "hav0c" won't work even though it could be considered as similar to "havoc"). It seems to only work as a "starting with" query (aka "h" or "ha" or "hav" etc. does the job), so I'm not sure about using the description from `fetch()`.. although the query param on that works in the same way, so that could also be changed in this PR

Reply:
Yeah the API doc is very specific about "starts with a provided string" so keep it that way.

Reply:
This *may* sound better, but better wording for this would be nicer 👍 

```suggestion
   * @property {string} options.query Search for members whose usernames or nicknames start with this
```

Reply:
Alternatively, what about this?
```suggestion
   * @property {string} options.query Filter members whose username or nickname start with the query.
```